x/ccv/consumer/types/params.go

```diff
@@ -1,15 +1,28 @@
 package types
 
 import (
-	"time"
+	fmt "fmt"
+	time "time"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	ccvtypes "github.com/cosmos/interchain-security/x/ccv/types"
 )
 
 const (
 	// about 2 hr at 7.6 seconds per blocks
 	DefaultBlocksPerDistributionTransmission = 1000
+
+	// 1 week
+	DefaultTransferTimeoutPeriod = 1 * 7 * 24 * time.Hour
```

Comment:
`DefaultTransferTimeoutPeriod` should match `DefaultBlocksPerDistributionTransmission`, see https://github.com/cosmos/interchain-security/issues/396#issuecomment-1278611737. Since IBC token transfers do not have to be in order, it could be easier to reason about the distribution protocol if the previous reward times out before sending the next one. Notice that on timeout, the transferred funds are added back to the pool, so the next transfer will include them as well.

Reply:
All makes sense and nice catch, I've made the default transfer timeout 1 hour now

Reply:
@jtremback Do you see any problems with this?